data/json/monstergroups/wilderness.json

```diff
@@ -2482,21 +2482,46 @@
       },
       {
         "monster": "mon_centipede_giant",
-        "freq": 40,
+        "freq": 45,
         "pack_size": [ 2, 3 ],
         "cost_multiplier": 2,
         "starts": 1440,
         "ends": 2160,
         "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
       },
+      {
+        "monster": "mon_centipede_mom",
+        "freq": 15,
+        "pack_size": [ 1, 2 ],
+        "cost_multiplier": 2,
+        "starts": 1440,
+        "ends": 2160,
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
       {
         "monster": "mon_centipede_giant",
-        "freq": 50,
+        "freq": 60,
         "cost_multiplier": 2,
         "pack_size": [ 2, 3 ],
         "starts": 2160,
         "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
       },
+      {
+        "monster": "mon_centipede_mom",
+        "freq": 20,
+        "cost_multiplier": 2,
+        "pack_size": [ 1, 3 ],
+        "starts": 2160,
+        "conditions": [ "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_centipede_mega",
+        "freq": 20,
+        "cost_multiplier": 2,
```

Comment:
Mega/Armored have the same cost multiplier as the others?

Reply:
For the swamp spawn I went with predators = 2, prey/nonthreatening mobs = 1, with the actual spawn scaling taken care of with the frequencies + time-to-spawn. Since swamps get 2-6 population, that means that theoretically 1-3 predator pods can spawn in a tile. Bumping them up to 3 would disqualify them from spawning in the tiles that rolled 2 population, and would be inconsistent with the rest.